### PR TITLE
fix(core-paragraph): fixed text/paragraph(large) line-height

### LINKED ISSUE #620: [Text] large text line height is too large
## Description
* Text (large) and paragraph (large) currently have a line-height of 2 (40px) when it should be 32px (as defined by DSM).

## Reproduction Steps
* Inspect large text or paragraph
* Observe "line-height: 2;"

## Reproduction Percentage (if applicable)
* 10/10 times

## Meta
* TDS version: v1.0.0
* Willing to develop solution: No
* Has workaround: No
* High impact: Kind of

## Screenshots
lineheight

### PR DESCRIPTION
**Pull request by derekkramer**
Fixed text (large) and paragraph (large), changing line-height of 2 (40px) to 32px to be compliant with DSM.

fixes #620

### CODE REVIEW

**Comment by andrewkumar** [comment]
Thanks @derekkramer 

Where are you using TDS out of curiosity?

**Comment by derekkramer** [comment]
I'm not actually using TDS personally, but I'm familiar with the organization and was glad to find an issue that I could help resolve.

**Comment by andrewkumar** [comment]
Thanks @derekkramer!

File: shared/styles/Typography.modules.scss [hunk 1]
```diff
@@ -58,7 +58,7 @@
 
   font-size: 1.25rem;
   letter-spacing: -1px;
-  line-height: 2;
+  line-height: 32px;
```
Comment (theetrain) [inline shared/styles/Typography.modules.scss]: Please change this to its unitless equivalent, `1.6`

**Pull request merged**